docs/self-hosted/deployment.md

```diff
@@ -70,6 +70,13 @@ configuration options to the ones listed under "Standard pooling".
 Using pool configuration overrides standard pool options. It is recommended to set both `pool_` and not `pool_` prefixed values to
 ensure that the standard pool options are set as well (`postgres://...?max_conns=4&pool_max_conns=4`).
 
+:::note
+
+To activate high-performance pooling, you must set at least the `pool_min_conns` parameter; otherwise, high-performance pooling
+will not be enabled.
+
+:::
+
```

Comment:
```suggestion
To activate high-performance pooling, you must set at least the `pool_min_conns` parameter; otherwise, high-performance pooling
will not be enabled.
```

Please don't use a note here - I understand it is tempting to do but most things shouldn't be an admonition

Reply:
sorry, didn't notice your comment, fix is here: https://github.com/ory/docs/pull/2155